deployment/network-operator/charts/node-feature-discovery/templates/master.yaml

```diff
@@ -58,17 +58,19 @@ spec:
                 fieldPath: spec.nodeName
           command:
             - "nfd-master"
-          
           resources:
             {{- toYaml .Values.master.resources | nindent 12 }}
+          args:
+            { { - if .Values.master.instance | empty | not } }
+            - "--instance={{ .Values.master.instance }}"
```

Comment:
I don't think we want it to be configurable. you can hardcode the value nvidia.networking or something else

Reply:
Let's hardcode it in values